src/lib.rs

```diff
@@ -579,7 +579,7 @@ struct private;
 #[cfg(feature = "parsing")]
 mod error;
 #[cfg(feature = "parsing")]
```

Comment:
Hmm, if we want these to be generally available, should they still stay behind the parsing feature? Perhaps a different feature might be relevant to use here.

Reply:
Good call. These types don't affect compile time so it should be okay to provide them even with `default-features = false`.